hudi-common/src/main/java/org/apache/hudi/common/table/timeline/HoodieDefaultTimeline.java

```diff
@@ -167,12 +167,13 @@ public HoodieDefaultTimeline getWriteTimeline() {
 
   @Override
   public HoodieTimeline getContiguousCompletedWriteTimeline() {
-    Option<HoodieInstant> earliestPending = getWriteTimeline().filterInflightsAndRequested().firstInstant();
+    HoodieDefaultTimeline writeTimeline = getWriteTimeline();
```

Comment:
is this now changing this method to look for contiguous timeline on both active + archived? whats the effect of this change

Reply:
Just a small optimization by hand, not relevent with this PR.